gulpfile.js

```diff
@@ -359,11 +359,7 @@ const criticalCssConfig = (function () {
 
             home: global.concat(
                 listing,
-                listingMenu,
-                /.*\.carousel.*/,
-                '.carousel__control--toggler',
-                '.carousel__items',
-                /\.carousel__item.*/
```

Comment:
do we need to modify this file?

Reply:
It's needed. But the styles here no longer exist. 

I've created another ticket for us to add in classes that maybe need to be done. As this will improve the experience for our users: https://github.com/elifesciences/issues/issues/7814